README.rst

```diff
@@ -49,3 +49,11 @@ Developers should follow the development install instructions found in the
 `documentation <https://cta-observatory.github
 .io/ctapipe/getting_started>`_.
 
+
+If you're installing *ctapipe* manually, you'll also need these external dependencies:
```

Comment:
I would rather give `pip installs here`, as `git clone` is not enough:

```
pip install https://github.com/cta-observatory/pyhessio/archive/master.tar.gz
pip install https://github.com/cta-observatory/ctapipe-extra/archive/master.tar.gz
pip install https://github.com/cta-sst-1m/protozfitsreader/archive/master.tar.gz
```

Reply:
I'd even like to give people who want to install *all optional dependencies* an easy and easy to remember way to do so.